modules/openapi-generator/src/main/resources/go/README.mustache

```diff
@@ -113,6 +113,22 @@ r, err := client.Service.Operation(auth, args)
 {{/isOAuth}}
 {{/authMethods}}
 
+## Documentation for Utility Methods
+
+Due to the fact that model structure members are all pointers, this package contains
+a number of utility functions to easily obtain pointers to values of basic types.
+Each of these functions takes a value of the given basic type and returns a pointer to it:
+
+* `Bool`
```

Comment:
I like this API decision, but I wonder if it would be better to prefix these with something to avoid conflict with other potentially imported types or models. Something like `PtrBool` or similar?

Reply:
Sounds like a good idea, I'll do that.